contracts/leverage_trading/src/metadata.rs

```diff
@@ -89,6 +90,8 @@ pub struct Order {
     pub block: BlockHeight,
     pub timestamp_ms: Timestamp,
     pub lpt_id: String,
+    /// data after closed position for trade history -> (Fee, PnL)
+    pub data_for_position_history: Option<(U128, PnLView)>,
```

Comment:
Rename it to history_data and create separate structure for it with fields fee, pnl

Reply:
Done